src/rules/noDuplicateImportsRule.ts

```diff
@@ -29,9 +31,13 @@ export class Rule extends Lint.Rules.AbstractRule {
         rationale: Lint.Utils.dedent`
             Using a single import statement per module will make the code clearer because you can see everything being imported
             from that module on one line.`,
-        optionsDescription: "Not configurable",
-        options: null,
-        optionExamples: [true],
+        optionsDescription: Lint.Utils.dedent`
+            "${OPTION_ALLOW_SEPARATE_NAMESPACE_IMPORTS}" allows you to import namespaces on separate lines.`,
+        options: {
+            type: "string",
```

Comment:
Although TSLint has traditionally had rule arguments provided as strings, that's starting to be a bit of a pain when they become more complex. Let's have the rule take in an object:

```ts
[true, {
    "allow-namespace-imports": true
}]
```

`memberOrderingRule.ts` is a good example of this.